instrumentation/jmx-metrics/javaagent/src/main/java/io/opentelemetry/instrumentation/javaagent/jmx/JmxMetricInsightInstaller.java

```diff
@@ -80,14 +80,15 @@ private static void buildFromDefaultRules(
 
   private static void buildFromUserRules(
       MetricConfiguration conf, ConfigProperties configProperties) {
-    String jmxDir = configProperties.getString("otel.jmx.config");
-    if (jmxDir != null) {
-      JmxMetricInsight.getLogger().log(FINE, "JMX config file name: {0}", jmxDir);
+    String files = configProperties.getString("otel.jmx.config", "");
+    String[] configFiles = files.isEmpty() ? new String[0] : files.split(",");
+    for (String configFile : configFiles) {
+      JmxMetricInsight.getLogger().log(FINE, "JMX config file name: {0}", configFile);
       RuleParser parserInstance = RuleParser.get();
-      try (InputStream inputStream = Files.newInputStream(new File(jmxDir.trim()).toPath())) {
-        parserInstance.addMetricDefsTo(conf, inputStream);
+      try (InputStream inputStream = Files.newInputStream(new File(configFile).toPath())) {
```

Comment:
```suggestion
      try (InputStream inputStream = Files.newInputStream(Paths.get(configFile))) {
```

Reply:
Thanks for the review, @mateuszrzeszutek!